packages/config/config/devices/0x019b/4512689.json

```diff
@@ -10,7 +10,12 @@
 			"productType": "0x0003",
 			"productId": "0x3601",
 			"zwaveAllianceId": 3933
-		}
+		},
+    {
+      "productType": "0x0003",
+      "productId": "0x000d",
+      "zwaveAllianceId": 3933
```

Comment:
@blhoward2 should we remove that line if the device ID does not match the one on the alliance webite?